dtfs-central-api/Dockerfile

```diff
@@ -5,10 +5,10 @@ FROM node:22.8.0-alpine3.20
 ARG GITHUB_SHA
 ENV GITHUB_SHA=$GITHUB_SHA
 
-# Node setup
+# Directories setup
 WORKDIR /app
 COPY package.json .
-COPY package-lock.json .
+COPY ../package-lock.json .
```

Comment:
Are we sure we want to jump up a dir? it seems that we're on the root of the project (due to line 14 `COPY dtfs-central-api dtfs-central-api`)

Reply:
Good spot, have resolved.